src/core/generator/KeepChangelogGenerator.ts

```
@@ -390,20 +462,404 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+      case 'use-tag-version': {
+        // Use the tag version for the new commits, preserve everything else
+        const versionEntry = entries.find(entry => entry.version !== 'Unreleased');
+        if (versionEntry && this.hasChanges(versionEntry)) {
+          const finalEntries = [...existingEntries];
+          finalEntries.unshift(versionEntry);
+          return finalEntries;
+        }
+        return existingEntries;
+      }
+
+      case 'no-conversion':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)